test/NuGet.Core.Tests/NuGet.Packaging.Core.Test/NuSpecCoreReaderTests.cs

```diff
@@ -38,17 +59,19 @@ public void GetPackageType_ReturnsEmptyPackageTypeListIfNotSpecifiedInManfiest()
    <somestuff>some-value</somestuff>
    <ver>123</ver>
   </metadata>
-</package>")]
+</package>")] // Elements with no children should be extracted.
         [InlineData(@"<?xml version=""1.0""?>
 <package xmlns=""a-random-xsd"">
   <metadata>
-   <packageType name=""Managed"" version=""2.0"" />
+   <packageTypes>
```

Comment:
Do we still support parsing the old way where there is one packageType node under metadata?